### DISCUSSION TITLE
SSH Tunnel is broken

### DISCUSSION THREAD

**Discussion by alpersamur3** [fasmide/remotemoe#13] [General]
Ssh tunnel is broken and not working after your last update. When I try to start the tunnel it gives the following error.
ssh: Could not resolve hostname remote.moe:No such host known
Please resolve this problem quickly.I'm using this tunnel self hosted database.

**Comment by fasmide** [comment]
Hehe yeah I've noticed, please refer to the [readme](https://github.com/fasmide/remotemoe#what-its-not) and also #12 

I have no idea when the remote.moe service will return - but I have tried to contact the relevant parties

**Reply by alpersamur3** [reply]
Ok bro.I'm waiting.
